usng4j-impl/src/main/java/org/codice/usng4j/impl/UsngCoordinateImpl.java

```diff
@@ -106,7 +106,31 @@
     this(zoneNumber, latitudeBandLetter, columnLetter, rowLetter);
     this.easting = easting;
     this.northing = northing;
-    this.precision = CoordinatePrecision.forEastNorth(easting, northing);
+    this.precision =
+        CoordinatePrecision.forEastNorth(Integer.toString(easting), Integer.toString(northing));
+  }
+
+  /**
+   * @param zoneNumber the zone number of this USNG coordinate.
+   * @param latitudeBandLetter the latitude band letter of this USNG coordinate.
+   * @param columnLetter the grid column letter of this USNG coordinate.
+   * @param rowLetter the grid row letter of this USNG coordinate.
+   * @param easting - the easting value of this USNG coordinate.
+   * @param northing - the northing value of this USNG coordinate.
+   * @param precision - the precision value of this USNG coordinate.
+   */
+  UsngCoordinateImpl(
+      final int zoneNumber,
+      final char latitudeBandLetter,
+      final char columnLetter,
+      char rowLetter,
+      int easting,
+      int northing,
+      CoordinatePrecision precision) {
+    this(zoneNumber, latitudeBandLetter, columnLetter, rowLetter);
+    this.easting = easting;
+    this.northing = northing;
+    this.precision = precision;
```

Comment:
are these lines formatted?

Reply:
Yes, that's from running the maven formatter.